symbols/arm9.yml

```diff
@@ -1375,10 +1408,20 @@ arm9:
       address:
         NA: 0x204CE90
         EU: 0x204D1C8
+        JP: 0x204D1F0
       description: |-
         Initializes the DUNGEON_*_LIST script variable values (IDs 0x4f-0x54).
         
         No params.
+    - name: ReturnInt1
```

Comment:
In the specific case of enemy spawn this address is called before the unused Mew check so that if this function actually returns false, the monster spawn always fails. So yes, it could be a debug switch of the type "toggle enemy spawn" used when debugging dungeons or something. It's also called in 2 other fuction which at a glance are also related to enemy spawn, and if it returns false some pieces of code are skipped.

For the other copies, I found 8 of them in total in the EU rom. 3 of them are referenced in some address tables in arm9.bin that probably contain other debug functions, 3 others are referenced by global pointers, while only this one and another one seem to be used in actual gameplay-related functions (the other one seems to disable recruiting if it returns false).

If you think this current information is enough we can rename it `MonsterSpawnsEnabled`, otherwise I'd just call it `ReturnTrue` and then maybe we'll rename it in the future when we better understand what the other copies do

Reply:
Just because all the "copies" have the same code doesn't mean we need to label them the same. If `MonsterSpawnsEnabled` is a good name for this particular address, then we should call it that. If the other ones do something different, they probably correspond to different debug flags, and we should give them different names.

Reply:
Then I think it's fine to rename it. How should I reference the fact that it's probably a debug switch in the description? Something like:
```
This function seems to be a debug switch that the developers may have used to disable the random enemy spawn. 
If it returned false, the call to SpawnMonster inside TrySpawnMonsterAndTickSpawnCounter would not be executed.
```
would be ok?

Reply:
Yeah that sounds good to me. Just make sure to mention that it always returns true somewhere in there.